package.json

```diff
@@ -26,5 +26,8 @@
   "bugs": {
     "url": "https://github.com/OmarIthawi/arabic-mathjax/issues"
   },
-  "homepage": "http://www.omardo.com/arabic-mathjax/index.html"
+  "homepage": "http://www.omardo.com/arabic-mathjax/index.html",
+  "dependencies": {
+    "gulp-uglifycss": "^1.1.0"
```

Comment:
@OmarIthawi it looks like this should be added as a devDependency, not a production dependency, no?

Reply:
That's right. The `package.json` is probably the worst part of this plugin. Any other ideas to improve it?